src/sections/SideBar/components/NavLinks/SideBarLinks.js

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemIcon from "@material-ui/core/ListItemIcon";
+import ListItemText from "@material-ui/core/ListItemText";
+import HomeOutlinedIcon from "@material-ui/icons/HomeOutlined";
+import PersonOutlineOutlinedIcon from "@material-ui/icons/PersonOutlineOutlined";
+import MailOutlineOutlinedIcon from "@material-ui/icons/MailOutlineOutlined";
+import NotesOutlinedIcon from "@material-ui/icons/NotesOutlined";
+import { createMuiTheme, ThemeProvider } from "@material-ui/core/styles";
+import Link from "@material-ui/core/Link";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+    maxWidth: 200,
+    backgroundColor: "rgb(61, 61, 61)",
+    color: "gray",
+    border: "1px solid black",
```

Comment:
Let's investigate and add a style rule so that we can define rules for sizes on px with material, for example: 


    border: `${theme.size.1} solid black`, 
or something alike

Reply:
The idea is to avoid having random sizes if not a defined o prefixed size for all the components